src/components/RehydrationProvider.js

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
 // SPDX-License-Identifier: MIT
 
-// Delays loading untill the store is rehydrated
+// Delays loading until the store is rehydrated
```

Comment:
Not sure why this is file is part of this PR

Reply:
ok. i'll take it out